test/functional/example_test.py

```diff
@@ -114,13 +111,13 @@ def setup_network(self):
         them to self.nodes, connect them and then sync."""
 
         self.setup_nodes()
+        self.connect_nodes(0, 1)
```

Comment:
Nitty: Is there a reason for moving these up here? Can you please move them back to where they were by default?